src/exit/tasks.py

```diff
@@ -0,0 +1,101 @@
+import logging
+
+from .clients import execution_client
+from .contracts import leverage_strategy_contract, ostoken_vault_escrow_contract
+from .graph import (
+    graph_get_allocators,
+    graph_get_leverage_positions,
+    graph_ostoken_exit_requests,
+)
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+
+def force_exits() -> None:
+    """Force handle ltv overflow. Trigger os token position exits"""
+    block = execution_client.eth.get_block('finalized')
+    logger.debug('Current block: %d', block['number'])
+
+    # force exit leverage positions
+    leverage_positions = graph_get_leverage_positions()
+    logger.info('Checking %d leverage positions...', len(leverage_positions))
+
+    # check by position borrow ltv
+    for position in leverage_positions:
+        if leverage_strategy_contract.can_force_enter_exit_queue(
+            vault=position.vault,
+            user=position.user,
+        ):
+            logger.info(
+                'Force exiting leverage positions: vault=%s, user=%s...',
+                position.vault,
+                position.user,
+            )
+            leverage_strategy_contract.force_enter_exit_queue(
```

Comment:
you must also check whether there is no pending exit requests and claim if there is one, otherwise this call will fail